linker/Linker/BCL.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Mono.Cecil;
+
+namespace Mono.Linker
+{
+	static class BCL
+	{
+		public static class EventTracingForWindows
+		{
+			public static bool IsEventSourceImplementation (TypeDefinition type, LinkContext context = null)
+			{
+				if (!type.IsClass)
+					return false;
+
+				while (type.BaseType != null) {
+					var bt = type.BaseType.Resolve ();
+
+					if (bt == null) {
+						if (context != null && !context.IgnoreUnresolved)
+							throw new ResolutionException (type.BaseType);
+					}
+
+					if (IsEventSourceType (bt))
```

Comment:
bt is null and `context.IgnoreUnresolved` is true, which it will be for us because we don't want to throw on unresolved, then the exception will be skipped.  However, it comes here and calls `IsEventSourceType(bt)` and `IsEventSourceType ` is going to throw because it was passed `null`

Reply:
Correct, pushed a fix for this